packages/simcore-sdk/src/simcore_sdk/node_ports_common/filemanager_utils.py

```diff
@@ -54,7 +54,7 @@ def _get_https_link_if_storage_secure(url: str) -> str:
     return url
 
 
-async def _complete_upload(
+async def complete_upload(
```

Comment:
not sure why all these suddenly become public functions?

Reply:
made public because they are used externally